JumpscaleCore/clients/gedis/GedisClient.py

```diff
@@ -51,7 +51,6 @@ def _init(self, **kwargs):
         self._redis_ = None
         self._threebot_me_ = None
         self._reset()
-        self.reload()
```

Comment:
Accessing self.actors will call the reload. We don't need to do this in the init, this makes it impossible to delete client instances for servers that are no longer running